packages/components/src/Heading/Heading.tsx

```diff
@@ -0,0 +1,44 @@
+import { TextProps, TypographyComponent } from "../Text/Text";
+import React from "react";
+import { styled } from "twin.macro";
+
+type HeadingElement = "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
+
+export type HeadingProps = TextProps & {
```

Comment:
Are we only exporting HeadingProps for storybook? If so, that's fine, but we could also infer them in the story

```ts
// Heading.stories.tsx

type Args = React.ComponentProps<typeof Heading>
```

Or if we want to use HeadingProps for other reasons, that's fine, too.